.github/workflows/tag-release.yml

```diff
@@ -1,31 +1,54 @@
 name: Release from tags
 # Create a GitHub release when a tag is pushed.
 
+# Input can ingest any string, but it will not work with arbitrary hashes. The code expects to get
+# a tag that sits on a release commit, so that commit is automatically skipped. Similarly, it
+# expects that the tag name conforms to a particular scheme.
+
 on:
   push:
     tags:
       # this is a glob, not a regexp
       - 'anaconda-*'
 
+  workflow_dispatch:
+    inputs:
+      tag-ref:
+        description: Tag to release from. Arbitrary hashes are not supported. Make sure to trim whitespace when copying.
+        type: string
+        required: true
+
 permissions:
   contents: write
 
 jobs:
   release-from-tag:
     runs-on: ubuntu-latest
-
     steps:
+      - name: Resolve tag reference
+        id: get_ref
+        # Get reference of the tag that is to be released.
+        # The tag can be in two places, depending on how the workflow was started.
+        run: |
+          if [ -n "${{ github.event.inputs.tag-ref }}" ] ; then
+            echo "Using manual input: ${{ github.event.inputs.tag-ref }}"
+            echo "::set-output name=ref::${{ github.event.inputs.tag-ref }}"
+          else
+            echo "Using automatic input: ${{ github.ref }}"
+            echo "::set-output name=ref::${{ github.ref }}"
+          fi
+
       - name: Extract version from the tag
         id: get_version
         # If the version does not fit the regexp, we'll just fail. TODO maybe abort instead?
         run: |
-          echo "Checking pushed tag ref ${{ github.ref }}"
-          VER=$(echo ${{ github.ref }} | perl -ne '/^refs\/tags\/anaconda-([0-9]+(?:\.[0-9]+){1,3})-1$/ && print "$1\n";')
+          echo "Checking ref ${{ steps.get_ref.outputs.ref }}"
+          VER=$(echo ${{ steps.get_ref.outputs.ref }} | perl -ne '/^(?:refs\/tags\/)?anaconda-([0-9]+(?:\.[0-9]+){1,3})-1$/ && print "$1\n";')
```

Comment:
Comment with what we are trying to get from what would be pretty helpful here.

Reply:
Agreed, I think this is rather information dense and a commit explaining how it works/what it does will help. :)

Reply:
I'm sorry, I do not understand what is the missing information so I can't add this comment. I guess I know this too well?